### DISCUSSION TITLE
error message when saving Page

### DISCUSSION THREAD

**Discussion by BSGMG18** [requarks/wiki#6069] [Error / Bug Report]
If i save a page once, I get the following error message.
"Cannot read properties of undefined (reading 'index')"

If i save the page a second time, I get this error message.
"Save conflict! Another user has already modified this page"

OS: Windows 10 Enterprise
Wiki.js version: 2.5.295
Database engine: PostgreSQL 15.1`
Node JS 16.16.0

**Comment by tilmanb** [comment]
> If i save a page once, I get the following error message. "Cannot read properties of undefined (reading 'index')"
> 
> If i save the page a second time, I get this error message. "Save conflict! Another user has already modified this page"
> 
> OS: Windows 10 Enterprise Wiki.js version: 2.5.295 Database engine: PostgreSQL 15.1` Node JS 16.16.0

We have the same error, also on Windows/PostgreSQL (Version 9) using any page editor.
The error appeared after some time, at first we did not get the error message, after some days it startet to appear.

The page is saved nevertheless, when you close the editor and open the page the new version is shown.

**Comment by WetSock** [comment]
I have the same problem

**Comment by wcadams55** [comment]
Same here. 

OS: Ubuntu 22.04 
Wiki.js version: 2.5.308 
Database engine: PostgreSQL 14.19
Node JS 12.22.9
